main.pm

```diff
@@ -424,8 +425,16 @@ sub load_consoletests() {
         loadtest "console/zypper_ref.pm";
         loadtest "console/yast2_lan.pm";
         loadtest "console/curl_https.pm";
+        if (!is_staging) {
+            # TEST DEVELOPERS - Put tests that you don't want to run in stagings below here
+            if (!get_var("OFW")) {
+                loadtest "console/aplay.pm";
+            }
+            loadtest "console/yast2_cmdline.pm";
+            loadtest "console/yast2_dns_server.pm";
+            loadtest "console/yast2_nfs_client.pm";
+        }
         if (!get_var("OFW")) {
-            loadtest "console/aplay.pm";
```

Comment:
yes - we retrigger failures of aplay openSUSE stagings (so far I remember once where it was actually a real issue, as retriggering did not make it go away...)